### ISSUE TITLE
Block Editor: Unable to Edit Text with Screen Reader

### ISSUE THREAD

**Issue by donalirl** [wordpress-mobile/gutenberg-mobile#2198]
### Expected behavior
I expect to be able to edit the text on posts and pages while using the app with a screen reader.

### Actual behavior
I cannot edit the text using the app with a screen reader.

Screen recording: https://d.pr/v/gO0VqP

### Steps to reproduce the behavior
1. Open a page or post in the app with paragraph blocks
2. Using only a screen reader, attempt to edit text.

**Comment by dcalhoun** [comment]
@donalirl thank you for reporting this issue! While exploring text editing using TalkBack, I definitely noticed aspects that are challenging. 

It would appear that original Droplr link you shared now results in a 404 error. Would you be able to provide a new video (possibly uploaded directly in a GitHub comment [now that it is supported](https://github.blog/2021-05-13-video-uploads-available-github/)) or (possibly even better) a list of specific issues you ran into while attempting to edit text with a screen reader? 

Thank you for your help!

**Comment by donalirl** [comment]
>It would appear that original Droplr link you shared now results in a 404 error. Would you be able to provide a new video (possibly uploaded directly in a GitHub comment now that it is supported) or (possibly even better) a list of specific issues you ran into while attempting to edit text with a screen reader?

Sorry about that! I didn't realize at the time that my droplr links were set to expire. I was able to locate the original video that demonstrates how I wasn't able to edit the paragraph block text:

https://user-images.githubusercontent.com/30727666/125214930-fe3d5380-e2fc-11eb-886d-ae21e36299d8.mp4

**Comment by hypest** [comment]
👋 all! Tried WPAndroid v17.8-rc-1 with Talkback enabled on a Pixel 2XL running Android 11 and seems that at this point, I'm able to edit paragraphs by double clicking the block.

Is the issue still happening for you in recent app versions @donalirl? Are there steps to reproduce perhaps? Thanks!

**Comment by donalirl** [comment]
Hi @hypest . I tried to reproduce just now and couldn't. I could double tap the block to edit the text. It seems that the issue can be closed.

Interestingly, I noticed Samsung/Android has an updated voice assistant tool (now called Talkback) so maybe some update in the past year solved this issue. Thanks for looking!
